frontend/apps/web/cypress/integration/Homepage.spec.ts

```diff
@@ -2,5 +2,23 @@ describe('Homepage spec', () => {
   it('Should check homepage', () => {
     cy.homepage()
     cy.get('h5.c-text--heading-5').should('contain', 'Systems')
+    cy.get('tbody>tr>td>div>span>a').eq(0).should('have.value', '')
```

Comment:
use comments for every line, we need to know by reading this what is it that this test is doing line by line.